src/components/course/CourseContextProvider.jsx

```diff
@@ -5,7 +5,6 @@ import {
 } from './data/constants';
 
 export const CourseContext = createContext();
-export const CourseContextConsumer = CourseContext.Consumer;
```

Comment:
this is not used anywhere, and can be accessed by `CourseContext.Consumer` anyways.